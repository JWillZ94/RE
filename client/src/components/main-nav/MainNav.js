import React from 'react';
import { BrowserRouter as Router, Route, NavLink } from 'react-router-dom';
import { Navbar, Nav, NavItem } from 'react-bootstrap';

import Home from '../home/Home';
import Buy from '../buy/Buy';
import Rent from '../rent/Rent';

const MainNav = () => (
  <Router>
    <Navbar className="">
      <Navbar.Header>
        <Navbar.Brand>
          <NavLink to="/" className="">RE Genie</NavLink>
        </Navbar.Brand>
      </Navbar.Header>
      <Nav>
        <NavItem>
          <NavLink to="/">Home</NavLink>
        </NavItem>
        <NavItem>
          <NavLink to="/buy">Buy</NavLink>
        </NavItem>
        <NavItem>
          <NavLink to="/rent" className="">Rent</NavLink>
        </NavItem>
      </Nav>

      <Route exact path="/" component={Home} />
      <Route path="/buy" component={Buy} />
      <Route path="/rent" component={Rent} />
    </Navbar>
  </Router>
);

export default MainNav;
