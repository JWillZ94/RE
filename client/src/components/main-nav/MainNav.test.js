import React from 'react';
import { mount, shallow, render } from 'enzyme';

import MainNav from './MainNav';

describe('MainNav', () => {
  it('should render', () => {
    const component = render(<MainNav />);

    expect(component).toMatchSnapshot();
  });
});
