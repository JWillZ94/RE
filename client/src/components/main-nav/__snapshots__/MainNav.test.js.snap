// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainNav should render 1`] = `
<nav
  class="navbar navbar-default"
>
  <div
    class="container"
  >
    <div
      class="navbar-header"
    >
      <a
        aria-current="page"
        class="navbar-brand active"
        href="/"
      >
        RE Genie
      </a>
    </div>
    <ul
      class="nav navbar-nav"
    >
      <li
        class=""
        role="presentation"
      >
        <a
          href="#"
          role="button"
        >
          
        </a>
        <a
          aria-current="page"
          class="active"
          href="/"
        >
          Home
        </a>
      </li>
      <li
        class=""
        role="presentation"
      >
        <a
          href="#"
          role="button"
        >
          
        </a>
        <a
          href="/buy"
        >
          Buy
        </a>
      </li>
      <li
        class=""
        role="presentation"
      >
        <a
          href="#"
          role="button"
        >
          
        </a>
        <a
          class=""
          href="/rent"
        >
          Rent
        </a>
      </li>
    </ul>
    <div
      class="bg-danger"
    >
      Homeeee
    </div>
  </div>
</nav>
`;
